lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -258,7 +281,7 @@ defmodule ExUnit.Formatter do
   defp test_location(msg, formatter), do: test_location(formatter.(:location_info, msg), nil)
 
   defp error_info(msg, nil) do
-    "     " <> String.replace(msg, "\n", "\n     ") <> <<"\n">>
+    "     " <> String.replace(msg, "\n", "\n     ") <> "\n"
```

Comment:
I think this change is not related to this PR (it's needed at some point of course though :smiley:).